testdroid/__init__.py

```diff
@@ -277,11 +277,28 @@ def get_me(self):
     def get_device_groups(self, limit=0):
         return self.get("me/device-groups", payload = {'limit': limit})
 
+    """ Returns list of frameworks
+    """
+    def get_frameworks(self, limit=0):
+        return self.get("me/available-frameworks", payload = {'limit': limit})
+
+    """ Returns list of project types
+    """
+    def get_project_types(self, limit=0):
+        return self.get("me/available-project-types", payload = {'limit': limit})
```

Comment:
Do we really need it? We want to remove this API resource in the future

Reply:
How about then a new project create, there you need to specify project type also so what could be an alternative way to get known about project types?